README.md

```diff
@@ -1,50 +1,74 @@
+
+> Changelogs are important to communicate end users what is going on with the project. See [keepachangelog.com](https://keepachangelog.com/en/1.0.0/) for additional reasoning.
+
+The changelog should be generated and updated automatically as part of the release process. [`lerna-changelog`](https://github.com/lerna/lerna-changelog) should be used to do so.
+
+> Lerna-changelog generates changelogs based on GitHub pull requests and their labels. Please refer to [it's documentation](https://github.com/lerna/lerna-changelog#lerna-changelog) for more details.
+
+lerna-changelog can be integrated with `release-it` to automatically generate the changelog as part of the release process. Robert Jackson (`@rwjblue`) provides a script to automate the setup for these tools: [create-rwjblue-release-it-setup](https://github.com/rwjblue/create-rwjblue-release-it-setup). It should be used for the setup and as a reference for recommended configuration.
+
+> Please refer to the [documentation of create-rwjblue-release-it-setup](https://github.com/rwjblue/create-rwjblue-release-it-setup#create-rwjblue-release-it-setup) for usage instructions. The script could be rerun on a repository already using lerna-changelog and release-it to update the configuration to the latest recommendations.
 
-* If your addon does not have a changelog, consider adding one
-	- Changelogs are important to communicate to end users what is going on with the project. See [keepachangelog.com](https://keepachangelog.com/en/1.0.0/) for additional reasoning.
-	- Robert Jackson (@rwjblue) has automated the initial setup of automatic changelog generation using some popular tools. Follow the steps in [create-rwjblue-release-it-setup](https://github.com/rwjblue/create-rwjblue-release-it-setup). You can see an example of how that works in [ember-pikaday](https://github.com/adopted-ember-addons/ember-pikaday).
+### GitHub labels
 
-* Use standard labels for issues and pull requests.
-	* Defer to the learning team's recommendations and [lerna-changelog](https://github.com/lerna/lerna-changelog)'s defaults.
+GitHub issues and pull request should be labeled. It should follow the [learning team's recommendations](https://github.com/emberjs/ember.js/blob/master/CONTRIBUTING.md#issue-labeling) and [lerna-changelog](https://github.com/lerna/lerna-changelog)'s defaults.
 
-* Add a code of conduct to the repo
-	- You can read how to add the Contributor Covenant here: [https://www.contributor-covenant.org/](https://www.contributor-covenant.org/)
-	
-## Hosting and Deployments
+### Code of conduct
 
-While we don't enforce a single way to host and deploy addon docs, many of the addons in this org use Netlify to deploy and host the addon docs.
+The [Ember Community Guidelines](https://emberjs.com/guidelines/) apply to all repositories within the org. It may be referenced in the repository's readme or in a `CODE_OF_CONDUCT.md`.
```

Comment:
This might be the most fundamental change to the meaning. I thought we should explicitly state that the code of conduct apply to all addons under the org regardless if it's explicitly mentioned in the repo. I also thought that it should point to ember community guidelines instead of the generic Contributor Covenant. But to be honest I'm not an expert on the topic and open to change back if this doesn't make much sense.

/cc @efx

Reply:
That is a great idea to defer to the community specific expectations; I had not considered that in detail. I'm with you not being an expert on the topic 😄